### DISCUSSION TITLE
How dijkstra_path works in networkx?

### DISCUSSION THREAD

**Discussion by TheDeathOMG** [networkx/networkx#5958] [Q&A]
Hello!

I am an student that is studying graphs and now I am learning the algorithm of Dijkstra to explain it. 

I obviously know how the _Algorithm of Dijkstra_ works, however I do not know how to portray my knowledge into a code. I searched a lot of stuff about this, but I couldn't find any information about how to do this algorithm right from the start using _networkx_. Is for that that I am searching about how is made the function that is implemented in this library. 

Thank you all for your attention!

**Comment by dschult** [comment]
The Python code is available on Github for you to see how we have implemented it in code. It is in the module `networkx/algorithms/shortest_paths/weighted.py`
